src/components/CommentInfo/CommentInfo.tsx

```diff
@@ -0,0 +1,18 @@
+import { Comment } from '../../types/Comment';
+import './CommentInfo.scss';
+
+type Props = {
+  comments: Comment[]
+};
+
+export const CommentInfo: React.FC<Props> = ({ comments }) => (
+  <ul className="comments">
+    {comments.map(comment => (
+      <li className="comments__info" key="comment.id">
```

Comment:
All the comments have key as a string "comment.id"
```suggestion
      <li className="comments__info" key={comment.id}>
```